src/utilities/dailyUsage.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2019
+ */
+
+import { UIDatabase } from '../database';
+import { MILLISECONDS_PER_DAY, millisecondsToDays } from '../database/utilities/index';
+
+const DEFAULT_LOOKBACK_PERIOD = 90;
+/**
+ * Returns either a customized lookback period, or 90 days in milliseconds.
+ */
+const getAMCLookbackPeriod = () => {
+  const amcLookbackString = UIDatabase.getSetting('monthlyConsumptionLookBackPeriod');
+  return (
+    (amcLookbackString ? Number(amcLookbackString) : DEFAULT_LOOKBACK_PERIOD) * MILLISECONDS_PER_DAY
+  );
+};
+
+/**
+ * Returns if this store is customized such that usage should force a lookback period,
+ * rather than using an items added date.
+ */
+const getAMCEnforcementLookback = () => {
+  const amcEnforcementString = UIDatabase.getSetting('monthlyConsumptionEnforceLookBackPeriod');
+  return amcEnforcementString === 'true';
+};
```

Comment:
this combined with the `checkModule` helper in the `ModuleReducer`.. lots of spread out helpers